setup.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import sys
+
+python_version = sys.version_info[:2]
+if python_version < (3, 6):
+    raise Exception(
+        f"This package requires Python 3.7 or greater. You have {python_version}."
+    )
+
+from setuptools import setup
+
+
+setup(
+    name="Mathics-doctest",
+    version="0.1.0",
+    install_requires=[
+        "Mathics3 >= 4.0.0"
```

Comment:
We don't want to install Mathics3 at all.  This would cause a circular dependency since mathics-core needs this.

In CI for full testing it can import mathics-core, but is that is not setup dependency. 

And locally and in CI for unit testing,  _data_ from core as it is can be pulled in here.  Some sort skeletal builtin modules can be created in a test directory.

But for to get started we can just try this without any tests.

Reply:
We also probably need a `packages` attribute in the setup to declare the module.

Reply:
> In CI for full testing it can import mathics-core, but is that is not a setup dependency.

Is it going to be done through just download or submodule? (I think the 1st is better in this case).

Reply:
As you suspect, download is simpler.